src/apis/operations.js

```diff
@@ -10,6 +10,17 @@ operations.resize = (width, height, options = {}) => {
   }
 }
 
+operations.autoRotate = (width, height, rotateDirection, options = {}) => {
```

Comment:
we named the function in other cases (dropshadow) as all lowercase, can you do that here too?